third_party/blink/renderer/platform/heap/thread_state_storage.h

```diff
@@ -43,12 +43,14 @@ struct ThreadingTrait {
 // Storage for all ThreadState objects. This includes the main-thread
 // ThreadState as well. Keep it outside the class so that PLATFORM_EXPORT
 // doesn't apply to it (otherwise, clang-cl complains).
-//extern thread_local ThreadStateStorage* g_thread_specific_ CONSTINIT
-//    __attribute__((tls_model(BLINK_HEAP_THREAD_LOCAL_MODEL)));
-
+#if BUILDFLAG(IS_WIN)
+extern thread_local ThreadStateStorage* g_thread_specific_ CONSTINIT
+    __attribute__((tls_model(BLINK_HEAP_THREAD_LOCAL_MODEL)));
+#else
```

Comment:
No `GetThreadStateStorage` in win?

Reply:
GetThreadStateStorage is a hack we have on linux to workaround thread_local not working correctly there.  thread_local works correctly on windows, and we should really fix things so that it works on linux too and remove these changes and the other places we have similar workarounds.